### ISSUE TITLE
More details in Project Settings view

### ISSUE THREAD

**Issue by dabeeeenster** [SolidStateGroup/bullet-train-api#3333]
### Is your feature request related to a problem? Please describe.

In the Project Settings view for a project, when you look at "Members", it is possible to distinguish between Regular Users and Project Administrators. However, you can not see the same for a group.

### Describe the solution you'd like.

When you look at the group list, we should add a column for whether the group is a Project Administrator or Regular Group.

In addition, it would be really helpful, in the same list view, to see for each user/group if that user/group have any project permissions (View, Create, Delete, Manage, etc.) without having the click on each user or group.

### Describe alternatives you've considered

None

**Comment by kyle-ssg** [comment]
> In the Project Settings view for a project, when you look at "Members", it is possible to distinguish between Regular Users and Project Administrators. However, you can not see the same for a group.

@dabeeeenster This isn't true, we only distinguish between organisation administrators and normal users but nothing about project permissions.

<img width="916" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/8608314/096473c4-b42f-49eb-9c78-259464c989d3">


I'm not sure whether we'd need to adjust the designs for this, would we want to show a list of permissions or perhaps Organisation Administrator | Project Administrator | Environment Administrator | x permissions / permission 1, permission 2

**Comment by dabeeeenster** [comment]
Yes I think here 
<img width="1118" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/173290/5ddac229-4fe4-4871-ac16-ead69e54fcee">
 
we should show a list of what permissions that group has.

**Comment by jatingodnani** [comment]
@matthewelwell is this issue open?

**Comment by kyle-ssg** [comment]
@jatingodnani absolutely, if you're offering to pick this up! I'll probably be picking this up in around a week if not.

**Comment by jatingodnani** [comment]
@kyle-ssg woulb you like to explain me more about this issue

**Comment by kyle-ssg** [comment]
@jatingodnani sure.

Essentially in the area Ben mentioned show the following if they are admin of that project/environment

```
Project Administrator
```

If they are not, we can show their permissions but truncate to the first 3 as tags e.g.

```
View Feature, Edit feature, Create Change request, +2 more 
```

**Comment by fabricanva** [comment]
Hi there! I noticed this open issue and I'm interested in working on it. Would it be possible for someone to assign it to me?

**Comment by dabeeeenster** [comment]
Great thanks - its done!

**Comment by fabricanva** [comment]
Thank you @dabeeeenster, I have submitted a PR for this: https://github.com/Flagsmith/flagsmith/pull/3629

**Comment by fabricanva** [comment]
I noticed that in the editPermission file, RTK query is not being used for certain requests (such as creating and updating a group). This could prevent the group and user lists from updating when such changes occur. Would it be advisable for me to create an issue regarding this?

**Comment by dabeeeenster** [comment]
> I noticed that in the editPermission file, RTK query is not being used for certain requests (such as creating and updating a group). This could prevent the group and user lists from updating when such changes occur. Would it be advisable for me to create an issue regarding this?

Q for @kyle-ssg and @novakzaballa !

**Comment by kyle-ssg** [comment]
@fabricanva you're exactly right, I have a couple of PRs in progress migrating things to RTK - related issue https://github.com/Flagsmith/flagsmith/issues/3619

**Comment by kyle-ssg** [comment]
Reopening as this was progressed in https://github.com/Flagsmith/flagsmith/pull/3629 (thank you so much) but still needs to be done for users

**Comment by fabricanva** [comment]
> Reopening as this was progressed in #3629 (thank you so much) but still needs to be done for users

No problem, I can implement it if you'd like @kyle-ssg 😁

**Comment by matthewelwell** [comment]
@fabricanva apologies that no one got back to you here. If you're still interested in doing this work, we'd love for you to take it on.
